SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -1971,18 +2013,50 @@ void CSU2TCLib::ThermalConductivitiesGY(){
     }
 
     /*--- Translational contribution to thermal conductivity ---*/
-    ThermalCond_tr    += (15.0/4.0)*kb*gam_i/denom_t;
+    if (denom_t != 0) ThermalCond_tr += (15.0/4.0)*kb*gam_i/denom_t;
 
     /*--- Translational contribution to thermal conductivity ---*/
-    if (RotationModes[iSpecies] != 0.0)
-      ThermalCond_tr  += kb*gam_i/denom_r;
+    if (RotationModes[iSpecies] != 0.0 && denom_r != 0) ThermalCond_tr  += kb*gam_i/denom_r;
 
     /*--- Vibrational-electronic contribution to thermal conductivity ---*/
-    ThermalCond_ve += kb*Cvve/R*gam_i / denom_r;
+    if (denom_r != 0) ThermalCond_ve += kb*Cvve/R*gam_i / denom_r;
   }
 
   ThermalConductivities[0] = ThermalCond_tr;
   ThermalConductivities[1] = ThermalCond_ve;
+}
+
+void CSU2TCLib::ViscositySuth(){
+
+  su2double T_ref = 273.15;
```

Comment:
You should define this as a constant somewhere.

Reply:
Done